### PR TITLE
rocketchat: add patches to rocketchat

### PR DESCRIPTION
**Pull request by BehindLoader**
Russian localization in our version of Rocket.Chat have some language errors. This patches are fixing it

### CODE REVIEW

**Comment by sibiryoff** [comment]
@BehindLoader @eugulixes shouldn't we update to [0.74.0](https://github.com/RocketChat/Rocket.Chat/pull/13474) instead?

**Comment by BehindLoader** [comment]
@sibiryoff This version does not include some of the changes that are in my pull request. I made a pull request in the official Rocket Chat repository (RocketChat/Rocket.Chat#14622), but it has not been reviewed yet.

**Comment by BehindLoader** [comment]
**Added patches to docker image building**
Patches now runs on image building. All files from directory `rocketchat/patches/` are applied on rocketchat build version

**Added patch for updating default permissions list**
Add rocketchat patch for permissions adding
There is some permissions which are added by this patch. These permissions will be used in hubot script named "hubot-viva-las-vegas" for differentiation of access rights of bot commands
- `hubot-viva-send-leave-request` - permission for allowing users to send leave request
- `hubot-viva-approve-leave-request` - permission for allowing users to approve leave requests
- `hubot-viva-reject-leave-request` - permission for allowing users to reject leave requests
- `hubot-viva-cancel-leave-request` - permission for allowing users to cancel leave requests
- `hubot-viva-get-leave-requests-list` - permission for allowing users to get list of users leave request statuses
- `hubot-viva-admin-create-leave-request` - permission for allowing users to create new leave request for another user without minimal date limits
- `hubot-viva-admin-create-time-off` - permission for allowing users to create time off requests for another user
- `hubot-viva-work-from-home` - permission for allowing users to use `работаю из дома` command
- `hubot-viva-illing` - permission for allowing users to use `болею` command

File: rocketchat/patches/hubot-viva-las-vegas-permissions.patch [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
++    // Canceling leave request
++    _id: 'hubot-viva-cancel-leave-request',
++    roles: ['']
++  }, {
++    // Getting leave requests list
++    _id: 'hubot-viva-get-leave-requests-list',
++    roles: ['']
++  }, {
++    // Initiating new leave request for another user
++    _id: 'hubot-viva-admin-create-leave-request',
```
Comment (eugulixes) [inline rocketchat/patches/hubot-viva-las-vegas-permissions.patch]: `hubot-viva-admin-initiate-new-leave-request-on-behalf-of-user`

File: rocketchat/patches/hubot-viva-las-vegas-permissions.patch [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
++    roles: ['']
++  }, {
++    // Initiates new time off for another user
++    _id: 'hubot-viva-admin-create-time-off',
```
Comment (eugulixes) [inline rocketchat/patches/hubot-viva-las-vegas-permissions.patch]: hubot-viva-admin-initiate-new-time-off-request-on-behalf-of-user

File: rocketchat/patches/hubot-viva-las-vegas-permissions.patch [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
++    roles: ['']
++  }, {
++    // Notifying about work from home
++    _id: 'hubot-viva-work-from-home',
++    roles: ['']
++  }, {
++    // Notifying about illing
++    _id: 'hubot-viva-illing',
```
Comment (eugulixes) [inline rocketchat/patches/hubot-viva-las-vegas-permissions.patch]: `hubot-viva-set-unset-status-of-being-ill`

File: rocketchat/Dockerfile [hunk 4]
```diff
@@ -8,12 +8,15 @@ ENV PUBLIC_KEY 0E163286C20D07B9787EBE9FD7F9D0414FD08104
...
 
+COPY patches /home/rocketchat/patches
+
 RUN apt-get update \
  && apt-get install -y \
     build-essential \
     curl \
     gnupg \
     python \
+    patch \
```
Comment (eugulixes) [inline rocketchat/Dockerfile]: Move up.

File: rocketchat/Dockerfile [hunk 5]
```diff
@@ -34,6 +42,7 @@ RUN apt-get update \
...
```
Comment (eugulixes) [inline rocketchat/Dockerfile]: Move up.

File: rocketchat/Dockerfile [hunk 6]
```diff
@@ -23,7 +26,12 @@ RUN apt-get update \
  && gpg --batch --verify rocket.chat.tgz.asc rocket.chat.tgz \
  && tar zxvf rocket.chat.tgz \
  && rm rocket.chat.tgz rocket.chat.tgz.asc \
- && cd $BUNDLE_DIR/programs/server \
+ && cd $BUNDLE_DIR \
+ # Patches apply
+ && for path in $(ls ../patches/) \
+ ; do echo "PATCH ${path}" && patch -p1 < "../patches/${path}" \
```
Comment (eugulixes) [inline rocketchat/Dockerfile]: Redundant `echo`. Remove it.

**Pull request merged**